main/glossary/README.md

```diff
@@ -337,10 +350,9 @@ Keys can be used as elements of CopySets and CopyBags and as keys of CopyMaps (s
 
```

Comment:
Above

> ```md
> [Values](#value) must be Keys.
> ```

Not in general. What this may originally have tried to say is that
- CopySets and CopyBags are Keys.
- CopyMaps are Keys iff all their values are Keys.

Reply:
Hmm, do you have an example of a non-Key Value?

Reply:
Sorry, let me elaborate... the full section currently reads like this:
> ## Value
> 
> Values are the part of an [amount](#amount) that describe the value of something
> that can be owned or shared: How much, how many, or a description of a unique asset, such as
> Pixel(3,2), $3, or “Seat J12 for the show September 27th at 9:00pm”.
> [Fungible](#fungible) values are usually represented by natural numbers.
> Other values may be represented as a CopySet of strings naming particular rights or
> arbitrary objects representing the rights directly (usually [non-fungible](#non-fungible) assets).
> Values must be [Keys](#key).
> 
> For more information, see the [ERTP documentation's Value section](/guides/ertp/amounts.md#values).

Are matchers/promises/errors/etc. allowed as the `value` part of an [Amount](https://docs.agoric.com/glossary/#amount)?

Reply:
Clarified elsewhere that the `Value` def above is `AmountValue`, i.e., the `value` part of an `Amount`. But when I said above

> CopyMaps are Keys iff all their values are Keys.

I meant "values" as in the values of a key-to-value mappings in a CopyMap.